dspace-api/src/main/java/org/dspace/content/dao/impl/CommunityDAOImpl.java

```diff
@@ -75,7 +76,7 @@ protected CommunityDAOImpl() {
             query.setMaxResults(limit);
         }
         query.setParameter(sortField.toString(), sortField.getID());
-        return list(query);
+        return new LinkedList<Community>(new LinkedHashSet<>(list(query)));
```

Comment:
As it is not immediately straightforward what is going on in this single line, I wonder if we should add a simple comment that says `// Convert List to HashSet and back again to filter out duplicates (e.g. a community having multiple titles/translations)`